2018-12-03T12:05:21.1526922+05:30  [FTL] Unable to start Kestrel. (fc6a528d)
System.IO.IOException: Failed to bind to address http://127.0.0.1:5000: address already in use. ---> Microsoft.AspNetCore.Server.Kestrel.Transport.Abstractions.Internal.AddressInUseException: Error -4091 EADDRINUSE address already in use ---> Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.Internal.Networking.UvException: Error -4091 EADDRINUSE address already in use
   at Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.Internal.Networking.LibuvFunctions.ThrowError(Int32 statusCode)
   at Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.Internal.Networking.LibuvFunctions.tcp_getsockname(UvTcpHandle handle, SockAddr& addr, Int32& namelen)
   at Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.Internal.Networking.UvTcpHandle.GetSockIPEndPoint()
   at Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.Internal.Listener.ListenTcp(Boolean useFileHandle)
   at Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.Internal.Listener.CreateListenSocket()
   at Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.Internal.Listener.<>c.<StartAsync>b__8_0(Listener listener)
   at Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.Internal.LibuvThread.CallbackAdapter`1.<>c.<.cctor>b__3_1(Object callback, Object state)
   at Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.Internal.LibuvThread.DoPostWork()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.Internal.ListenerPrimary.<StartAsync>d__15.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.LibuvTransport.<BindAsync>d__20.MoveNext()
   --- End of inner exception stack trace ---
   at Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.LibuvTransport.<BindAsync>d__20.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Server.Kestrel.Core.KestrelServer.<>c__DisplayClass21_0`1.<<StartAsync>g__OnBind0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.AddressBinder.<BindEndpointAsync>d__5.MoveNext()
   --- End of inner exception stack trace ---
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.AddressBinder.<BindEndpointAsync>d__5.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.AddressBinder.<BindLocalhostAsync>d__6.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.AddressBinder.DefaultAddressStrategy.<BindAsync>d__0.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.AddressBinder.<BindAsync>d__0.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Server.Kestrel.Core.KestrelServer.<StartAsync>d__21`1.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Hosting.Internal.WebHost.<StartAsync>d__26.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Hosting.WebHostExtensions.<RunAsync>d__5.MoveNext()
2018-12-03T12:13:47.0056020+05:30 0HLIOTAM6GCK3:00000001 [INF] Request starting HTTP/1.1 GET http://localhost:5000/   (ca22a1cb)
2018-12-03T12:13:47.0535534+05:30 0HLIOTAM6GCK3:00000001 [INF] Sending file. Request path: '"/index.html"'. Physical path: '"E:\Maran\Dodos-core\Dodos.UI\wwwroot\index.html"' (27b0a520)
2018-12-03T12:13:47.1480569+05:30 0HLIOTAM6GCK3:00000001 [INF] Request finished in 143.254ms 200 text/html (791a596a)
2018-12-03T12:14:12.3612606+05:30 0HLIOTAM6GCK3:00000002 [INF] Request starting HTTP/1.1 GET http://localhost:5000/Employee/GetAllEmployees   (ca22a1cb)
2018-12-03T12:14:13.0578102+05:30 0HLIOTAM6GCK3:00000002 [INF] Request finished in 696.4737ms 200 text/html (791a596a)
2018-12-03T12:14:51.6223430+05:30 0HLIOTAM6GCK3:00000003 [INF] Request starting HTTP/1.1 GET http://localhost:5000/Employee/GetAllEmployees   (ca22a1cb)
2018-12-03T12:14:51.6237937+05:30 0HLIOTAM6GCK3:00000003 [INF] Request finished in 1.4502ms 200 text/html (791a596a)
2018-12-03T12:14:51.6392931+05:30 0HLIOTAM6GCK3:00000004 [INF] Request starting HTTP/1.1 GET http://localhost:5000/favicon.ico   (ca22a1cb)
2018-12-03T12:14:51.6397379+05:30 0HLIOTAM6GCK3:00000004 [INF] Sending file. Request path: '"/favicon.ico"'. Physical path: '"E:\Maran\Dodos-core\Dodos.UI\wwwroot\favicon.ico"' (27b0a520)
2018-12-03T12:14:51.6625689+05:30 0HLIOTAM6GCK3:00000004 [INF] Request finished in 23.2632ms 200 image/x-icon (791a596a)
2018-12-03T12:14:53.0937529+05:30 0HLIOTAM6GCK3:00000005 [INF] Request starting HTTP/1.1 GET http://localhost:5000/Employee/GetAllEmployees   (ca22a1cb)
2018-12-03T12:14:53.0944080+05:30 0HLIOTAM6GCK3:00000005 [INF] Request finished in 0.6905ms 200 text/html (791a596a)
2018-12-03T12:14:53.1088199+05:30 0HLIOTAM6GCK3:00000006 [INF] Request starting HTTP/1.1 GET http://localhost:5000/favicon.ico   (ca22a1cb)
2018-12-03T12:14:53.1090766+05:30 0HLIOTAM6GCK3:00000006 [INF] Sending file. Request path: '"/favicon.ico"'. Physical path: '"E:\Maran\Dodos-core\Dodos.UI\wwwroot\favicon.ico"' (27b0a520)
2018-12-03T12:14:53.1095803+05:30 0HLIOTAM6GCK3:00000006 [INF] Request finished in 0.7873ms 200 image/x-icon (791a596a)
2018-12-03T12:14:53.2854906+05:30 0HLIOTAM6GCK3:00000007 [INF] Request starting HTTP/1.1 GET http://localhost:5000/Employee/GetAllEmployees   (ca22a1cb)
2018-12-03T12:14:53.2861476+05:30 0HLIOTAM6GCK3:00000007 [INF] Request finished in 0.6249ms 200 text/html (791a596a)
2018-12-03T12:14:53.2957270+05:30 0HLIOTAM6GCK3:00000008 [INF] Request starting HTTP/1.1 GET http://localhost:5000/favicon.ico   (ca22a1cb)
2018-12-03T12:14:53.2960229+05:30 0HLIOTAM6GCK3:00000008 [INF] Sending file. Request path: '"/favicon.ico"'. Physical path: '"E:\Maran\Dodos-core\Dodos.UI\wwwroot\favicon.ico"' (27b0a520)
2018-12-03T12:14:53.2962386+05:30 0HLIOTAM6GCK3:00000008 [INF] Request finished in 0.5317ms 200 image/x-icon (791a596a)
2018-12-03T12:14:53.4453641+05:30 0HLIOTAM6GCK3:00000009 [INF] Request starting HTTP/1.1 GET http://localhost:5000/Employee/GetAllEmployees   (ca22a1cb)
2018-12-03T12:14:53.4458914+05:30 0HLIOTAM6GCK3:00000009 [INF] Request finished in 0.5471ms 200 text/html (791a596a)
2018-12-03T12:14:53.4589174+05:30 0HLIOTAM6GCK3:0000000A [INF] Request starting HTTP/1.1 GET http://localhost:5000/favicon.ico   (ca22a1cb)
2018-12-03T12:14:53.4591565+05:30 0HLIOTAM6GCK3:0000000A [INF] Sending file. Request path: '"/favicon.ico"'. Physical path: '"E:\Maran\Dodos-core\Dodos.UI\wwwroot\favicon.ico"' (27b0a520)
2018-12-03T12:14:53.4594555+05:30 0HLIOTAM6GCK3:0000000A [INF] Request finished in 0.5499ms 200 image/x-icon (791a596a)
2018-12-03T12:14:53.6176393+05:30 0HLIOTAM6GCK3:0000000B [INF] Request starting HTTP/1.1 GET http://localhost:5000/Employee/GetAllEmployees   (ca22a1cb)
2018-12-03T12:14:53.6183441+05:30 0HLIOTAM6GCK3:0000000B [INF] Request finished in 0.67ms 200 text/html (791a596a)
2018-12-03T12:14:53.6306250+05:30 0HLIOTAM6GCK3:0000000C [INF] Request starting HTTP/1.1 GET http://localhost:5000/favicon.ico   (ca22a1cb)
2018-12-03T12:14:53.6308696+05:30 0HLIOTAM6GCK3:0000000C [INF] Sending file. Request path: '"/favicon.ico"'. Physical path: '"E:\Maran\Dodos-core\Dodos.UI\wwwroot\favicon.ico"' (27b0a520)
2018-12-03T12:14:53.6310702+05:30 0HLIOTAM6GCK3:0000000C [INF] Request finished in 0.4511ms 200 image/x-icon (791a596a)
2018-12-03T12:15:24.2865066+05:30 0HLIOTAM6GCK3:0000000D [INF] Request starting HTTP/1.1 GET http://localhost:5000/Employee   (ca22a1cb)
2018-12-03T12:15:24.2872913+05:30 0HLIOTAM6GCK3:0000000D [INF] Request finished in 0.7897ms 200 text/html (791a596a)
2018-12-03T12:16:02.6943457+05:30 0HLIOTAM6GCK7:00000001 [INF] Request starting HTTP/1.1 GET http://localhost:5000/Employee   (ca22a1cb)
2018-12-03T12:16:02.6950853+05:30 0HLIOTAM6GCK7:00000001 [INF] Request finished in 0.7379ms 200 text/html (791a596a)
2018-12-03T12:16:36.7854311+05:30 0HLIOTAM6GCK7:00000002 [INF] Request starting HTTP/1.1 GET http://localhost:5000/Employee/GetAllEmployees   (ca22a1cb)
2018-12-03T12:16:36.7862702+05:30 0HLIOTAM6GCK7:00000002 [INF] Request finished in 0.8529ms 200 text/html (791a596a)
2018-12-03T12:22:12.4119284+05:30 0HLIOTFAL9461:00000001 [INF] Request starting HTTP/1.1 GET http://localhost:5000/Employee/GetAllEmployees   (ca22a1cb)
2018-12-03T12:22:12.5008604+05:30 0HLIOTFAL9461:00000001 [INF] Request finished in 94.5678ms 200 text/html (791a596a)
2018-12-03T12:22:23.1000052+05:30 0HLIOTFAL9461:00000002 [INF] Request starting HTTP/1.1 GET http://localhost:5000/Employee/GetAllEmployees   (ca22a1cb)
2018-12-03T12:22:23.1014911+05:30 0HLIOTFAL9461:00000002 [INF] Request finished in 1.4968ms 200 text/html (791a596a)
2018-12-03T12:22:23.1175416+05:30 0HLIOTFAL9461:00000003 [INF] Request starting HTTP/1.1 GET http://localhost:5000/favicon.ico   (ca22a1cb)
2018-12-03T12:22:23.1188694+05:30 0HLIOTFAL9461:00000003 [INF] Sending file. Request path: '"/favicon.ico"'. Physical path: '"E:\Maran\Dodos-core\Dodos.UI\wwwroot\favicon.ico"' (27b0a520)
2018-12-03T12:22:23.1241881+05:30 0HLIOTFAL9461:00000003 [INF] Request finished in 6.603ms 200 image/x-icon (791a596a)
2018-12-03T14:36:33.5394341+05:30 0HLIOVOT0HB4H:00000001 [INF] Request starting HTTP/1.1 GET http://localhost:5000/Login application/json  (ca22a1cb)
2018-12-03T14:36:34.3862411+05:30 0HLIOVOT0HB4H:00000001 [INF] Request finished in 853.3815ms 200 text/html (791a596a)
2018-12-03T14:37:07.1518602+05:30 0HLIOVOT0HB4H:00000002 [INF] Request starting HTTP/1.1 POST http://localhost:5000/Login application/json 79 (ca22a1cb)
2018-12-03T14:37:07.1773286+05:30 0HLIOVOT0HB4H:00000002 [INF] Request finished in 25.4629ms 200 text/html (791a596a)
2018-12-03T14:37:08.2247451+05:30 0HLIOVOT0HB4H:00000003 [INF] Request starting HTTP/1.1 POST http://localhost:5000/Login application/json 79 (ca22a1cb)
2018-12-03T14:37:08.2261566+05:30 0HLIOVOT0HB4H:00000003 [INF] Request finished in 1.4589ms 200 text/html (791a596a)
2018-12-03T14:39:08.0767499+05:30 0HLIOVOT0HB4H:00000004 [INF] Request starting HTTP/1.1 POST http://localhost:5000/Login application/json 80 (ca22a1cb)
2018-12-03T14:39:08.0787224+05:30 0HLIOVOT0HB4H:00000004 [INF] Request finished in 1.7489ms 200 text/html (791a596a)
2018-12-03T15:06:16.7425835+05:30 0HLIP0B1FMRRP:00000001 [INF] Request starting HTTP/1.1 POST http://localhost:5000/Login application/json 80 (ca22a1cb)
2018-12-03T15:06:16.9204311+05:30 0HLIP0B1FMRRP:00000001 [INF] Request finished in 185.3873ms 200 text/html (791a596a)
2018-12-03T15:20:14.0334259+05:30 0HLIP0IM8R9HV:00000001 [INF] Request starting HTTP/1.1 GET http://localhost:5000/Login   (ca22a1cb)
2018-12-03T15:20:14.2211113+05:30 0HLIP0IM8R9HV:00000001 [INF] Request finished in 195.3425ms 200 text/html (791a596a)
2018-12-03T15:25:37.6003282+05:30 0HLIP0LSVE0LJ:00000001 [INF] Request starting HTTP/1.1 POST http://localhost:5000/Login application/json 80 (ca22a1cb)
2018-12-03T15:25:37.6909517+05:30 0HLIP0LSVE0LJ:00000001 [INF] Request finished in 97.0859ms 200 text/html (791a596a)
2018-12-03T15:26:22.5526742+05:30  [FTL] Unable to start Kestrel. (fc6a528d)
System.IO.IOException: Failed to bind to address http://127.0.0.1:5000: address already in use. ---> Microsoft.AspNetCore.Server.Kestrel.Transport.Abstractions.Internal.AddressInUseException: Error -4091 EADDRINUSE address already in use ---> Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.Internal.Networking.UvException: Error -4091 EADDRINUSE address already in use
   at Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.Internal.Networking.LibuvFunctions.ThrowError(Int32 statusCode)
   at Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.Internal.Networking.LibuvFunctions.tcp_getsockname(UvTcpHandle handle, SockAddr& addr, Int32& namelen)
   at Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.Internal.Networking.UvTcpHandle.GetSockIPEndPoint()
   at Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.Internal.Listener.ListenTcp(Boolean useFileHandle)
   at Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.Internal.Listener.CreateListenSocket()
   at Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.Internal.Listener.<>c.<StartAsync>b__8_0(Listener listener)
   at Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.Internal.LibuvThread.CallbackAdapter`1.<>c.<.cctor>b__3_1(Object callback, Object state)
   at Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.Internal.LibuvThread.DoPostWork()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.Internal.ListenerPrimary.<StartAsync>d__15.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.LibuvTransport.<BindAsync>d__20.MoveNext()
   --- End of inner exception stack trace ---
   at Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.LibuvTransport.<BindAsync>d__20.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Server.Kestrel.Core.KestrelServer.<>c__DisplayClass21_0`1.<<StartAsync>g__OnBind0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.AddressBinder.<BindEndpointAsync>d__5.MoveNext()
   --- End of inner exception stack trace ---
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.AddressBinder.<BindEndpointAsync>d__5.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.AddressBinder.<BindLocalhostAsync>d__6.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.AddressBinder.DefaultAddressStrategy.<BindAsync>d__0.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.AddressBinder.<BindAsync>d__0.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Server.Kestrel.Core.KestrelServer.<StartAsync>d__21`1.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Hosting.Internal.WebHost.<StartAsync>d__26.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Hosting.WebHostExtensions.<RunAsync>d__5.MoveNext()
2018-12-03T15:36:48.0483110+05:30  [FTL] Unable to start Kestrel. (fc6a528d)
System.IO.IOException: Failed to bind to address http://127.0.0.1:5000: address already in use. ---> Microsoft.AspNetCore.Server.Kestrel.Transport.Abstractions.Internal.AddressInUseException: Error -4091 EADDRINUSE address already in use ---> Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.Internal.Networking.UvException: Error -4091 EADDRINUSE address already in use
   at Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.Internal.Networking.LibuvFunctions.ThrowError(Int32 statusCode)
   at Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.Internal.Networking.LibuvFunctions.tcp_getsockname(UvTcpHandle handle, SockAddr& addr, Int32& namelen)
   at Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.Internal.Networking.UvTcpHandle.GetSockIPEndPoint()
   at Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.Internal.Listener.ListenTcp(Boolean useFileHandle)
   at Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.Internal.Listener.CreateListenSocket()
   at Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.Internal.Listener.<>c.<StartAsync>b__8_0(Listener listener)
   at Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.Internal.LibuvThread.CallbackAdapter`1.<>c.<.cctor>b__3_1(Object callback, Object state)
   at Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.Internal.LibuvThread.DoPostWork()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.Internal.ListenerPrimary.<StartAsync>d__15.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.LibuvTransport.<BindAsync>d__20.MoveNext()
   --- End of inner exception stack trace ---
   at Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.LibuvTransport.<BindAsync>d__20.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Server.Kestrel.Core.KestrelServer.<>c__DisplayClass21_0`1.<<StartAsync>g__OnBind0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.AddressBinder.<BindEndpointAsync>d__5.MoveNext()
   --- End of inner exception stack trace ---
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.AddressBinder.<BindEndpointAsync>d__5.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.AddressBinder.<BindLocalhostAsync>d__6.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.AddressBinder.DefaultAddressStrategy.<BindAsync>d__0.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.AddressBinder.<BindAsync>d__0.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Server.Kestrel.Core.KestrelServer.<StartAsync>d__21`1.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Hosting.Internal.WebHost.<StartAsync>d__26.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Hosting.WebHostExtensions.<RunAsync>d__5.MoveNext()
2018-12-03T15:42:58.1361098+05:30  [FTL] Unable to start Kestrel. (fc6a528d)
System.IO.IOException: Failed to bind to address http://127.0.0.1:5000: address already in use. ---> Microsoft.AspNetCore.Server.Kestrel.Transport.Abstractions.Internal.AddressInUseException: Error -4091 EADDRINUSE address already in use ---> Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.Internal.Networking.UvException: Error -4091 EADDRINUSE address already in use
   at Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.Internal.Networking.LibuvFunctions.ThrowError(Int32 statusCode)
   at Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.Internal.Networking.LibuvFunctions.tcp_getsockname(UvTcpHandle handle, SockAddr& addr, Int32& namelen)
   at Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.Internal.Networking.UvTcpHandle.GetSockIPEndPoint()
   at Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.Internal.Listener.ListenTcp(Boolean useFileHandle)
   at Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.Internal.Listener.CreateListenSocket()
   at Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.Internal.Listener.<>c.<StartAsync>b__8_0(Listener listener)
   at Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.Internal.LibuvThread.CallbackAdapter`1.<>c.<.cctor>b__3_1(Object callback, Object state)
   at Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.Internal.LibuvThread.DoPostWork()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.Internal.ListenerPrimary.<StartAsync>d__15.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.LibuvTransport.<BindAsync>d__20.MoveNext()
   --- End of inner exception stack trace ---
   at Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.LibuvTransport.<BindAsync>d__20.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Server.Kestrel.Core.KestrelServer.<>c__DisplayClass21_0`1.<<StartAsync>g__OnBind0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.AddressBinder.<BindEndpointAsync>d__5.MoveNext()
   --- End of inner exception stack trace ---
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.AddressBinder.<BindEndpointAsync>d__5.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.AddressBinder.<BindLocalhostAsync>d__6.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.AddressBinder.DefaultAddressStrategy.<BindAsync>d__0.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.AddressBinder.<BindAsync>d__0.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Server.Kestrel.Core.KestrelServer.<StartAsync>d__21`1.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Hosting.Internal.WebHost.<StartAsync>d__26.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Hosting.WebHostExtensions.<RunAsync>d__5.MoveNext()
2018-12-03T16:55:25.0687179+05:30 0HLIP282IBM1G:00000001 [INF] Request starting HTTP/1.1 POST http://localhost:5000/Login application/json 80 (ca22a1cb)
2018-12-03T16:55:25.1595691+05:30 0HLIP282IBM1G:00000001 [INF] Request finished in 97.0666ms 200 text/html (791a596a)
2018-12-03T18:05:51.9725935+05:30  [FTL] Unable to start Kestrel. (fc6a528d)
System.IO.IOException: Failed to bind to address http://127.0.0.1:5000: address already in use. ---> Microsoft.AspNetCore.Server.Kestrel.Transport.Abstractions.Internal.AddressInUseException: Error -4091 EADDRINUSE address already in use ---> Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.Internal.Networking.UvException: Error -4091 EADDRINUSE address already in use
   at Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.Internal.Networking.LibuvFunctions.ThrowError(Int32 statusCode)
   at Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.Internal.Networking.LibuvFunctions.tcp_getsockname(UvTcpHandle handle, SockAddr& addr, Int32& namelen)
   at Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.Internal.Networking.UvTcpHandle.GetSockIPEndPoint()
   at Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.Internal.Listener.ListenTcp(Boolean useFileHandle)
   at Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.Internal.Listener.CreateListenSocket()
   at Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.Internal.Listener.<>c.<StartAsync>b__8_0(Listener listener)
   at Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.Internal.LibuvThread.CallbackAdapter`1.<>c.<.cctor>b__3_1(Object callback, Object state)
   at Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.Internal.LibuvThread.DoPostWork()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.Internal.ListenerPrimary.<StartAsync>d__15.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.LibuvTransport.<BindAsync>d__20.MoveNext()
   --- End of inner exception stack trace ---
   at Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.LibuvTransport.<BindAsync>d__20.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Server.Kestrel.Core.KestrelServer.<>c__DisplayClass21_0`1.<<StartAsync>g__OnBind0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.AddressBinder.<BindEndpointAsync>d__5.MoveNext()
   --- End of inner exception stack trace ---
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.AddressBinder.<BindEndpointAsync>d__5.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.AddressBinder.<BindLocalhostAsync>d__6.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.AddressBinder.DefaultAddressStrategy.<BindAsync>d__0.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.AddressBinder.<BindAsync>d__0.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Server.Kestrel.Core.KestrelServer.<StartAsync>d__21`1.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Hosting.Internal.WebHost.<StartAsync>d__26.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Hosting.WebHostExtensions.<RunAsync>d__5.MoveNext()
2018-12-03T19:23:08.8045217+05:30 0HLIP4QAQ8QNQ:00000001 [INF] Request starting HTTP/1.1 GET http://localhost:5000/Login/Postcheck   (ca22a1cb)
2018-12-03T19:23:09.3194666+05:30 0HLIP4QAQ8QNQ:00000001 [INF] Request finished in 545.6811ms 200 text/html (791a596a)
2018-12-03T19:23:31.5106137+05:30 0HLIP4QAQ8QNQ:00000002 [INF] Request starting HTTP/1.1 GET http://localhost:5000/Login/Postcheck   (ca22a1cb)
2018-12-03T19:23:31.5116194+05:30 0HLIP4QAQ8QNQ:00000002 [INF] Request finished in 1.0046ms 200 text/html (791a596a)
2018-12-03T19:23:31.5349565+05:30 0HLIP4QAQ8QNQ:00000003 [INF] Request starting HTTP/1.1 GET http://localhost:5000/favicon.ico   (ca22a1cb)
2018-12-03T19:23:31.5536093+05:30 0HLIP4QAQ8QNQ:00000003 [INF] Sending file. Request path: '"/favicon.ico"'. Physical path: '"E:\Maran\Dodos-core\Dodos.UI\wwwroot\favicon.ico"' (27b0a520)
2018-12-03T19:23:31.5695200+05:30 0HLIP4QAQ8QNQ:00000003 [INF] Request finished in 34.5884ms 200 image/x-icon (791a596a)
2018-12-03T19:25:24.9699928+05:30 0HLIP4RSNDHA9:00000001 [INF] Request starting HTTP/1.1 GET http://localhost:5000/Login/Postcheck   (ca22a1cb)
2018-12-03T19:25:25.2609801+05:30 0HLIP4RSNDHA9:00000001 [INF] Request finished in 300.4826ms 200 text/html (791a596a)
2018-12-03T19:25:25.2730832+05:30 0HLIP4RSNDHA9:00000002 [INF] Request starting HTTP/1.1 GET http://localhost:5000/favicon.ico   (ca22a1cb)
2018-12-03T19:25:25.2851362+05:30 0HLIP4RSNDHA9:00000002 [INF] Sending file. Request path: '"/favicon.ico"'. Physical path: '"E:\Maran\Dodos-core\Dodos.UI\wwwroot\favicon.ico"' (27b0a520)
2018-12-03T19:25:25.3007724+05:30 0HLIP4RSNDHA9:00000002 [INF] Request finished in 27.6574ms 200 image/x-icon (791a596a)
2018-12-03T19:25:48.9607697+05:30 0HLIP4RSNDHA9:00000003 [INF] Request starting HTTP/1.1 GET http://localhost:5000/Login/   (ca22a1cb)
2018-12-03T19:25:48.9618941+05:30 0HLIP4RSNDHA9:00000003 [INF] Request finished in 1.1622ms 200 text/html (791a596a)
